subprojects/docs/src/docs/release/notes.md

```diff
@@ -71,6 +71,10 @@ It is now possible to compile native applications with the Visual C++ toolchain
 With this release, Gradle will also begin discovering Visual Studio installations using the [vswhere utility](https://github.com/Microsoft/vswhere)
 from Microsoft if it's available.  
 
+### Embedded Ant version upgraded to Ant 1.9.9
+
+Gradle now embeds [Ant 1.9.9](https://archive.apache.org/dist/ant/RELEASE-NOTES-1.9.9.html). Previous releases used Ant 1.9.6.
```

Comment:
I think the only concern I'd have with this upgrade would be consumers that rely on some API that the new Ant version might break. At the moment I have no idea how aggressive Ant is in that regard. I guess we'll see after publishing the RC. Hopefully, users will tell us.

Reply:
I didn't see anything alarming from the release notes.  1.9.7 and 1.9.8 added support for JDK9, so it seems like something we should do anyways.